### PR TITLE
Push images for different dropins sets

### LINKED ISSUE #134: Add public Capella images with dropins included
Many Capella models depend on specific dropins. 

We should offer three sets of dropins for all Capella based images: 
- `none` - No dropins are included (how it is handled currently)
- `selected` - The following dropins are included: 
  - [System to Subsystem Transition](https://github.com/eclipse/capella-sss-transition)
  - [XHTML Documentation Generation](https://github.com/eclipse/capella-xhtml-docgen)
  - [Filtering](https://github.com/eclipse/capella-filtering)
  - [Property Values Management Tools (PVMT)](https://github.com/eclipse/capella/wiki/PVMT)
  - [Requirements Viewpoint](https://github.com/eclipse/capella-requirements-vp)
  - [Textual Editor](https://github.com/eclipse/capella-textual-editor)
- `all` - All open source dropins (with EPL license): https://www.eclipse.org/capella/addons.html

### PR DESCRIPTION
**Pull request by MoritzWeber0**
Resolves #134

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,10 +71,14 @@ METRICS_PORT ?= 9118
 export CAPELLA_VERSIONS ?= 5.0.0 5.2.0 6.0.0
 
 # Capella version used to run containers
-CAPELLA_VERSION ?= 6.0.0
+export CAPELLA_VERSION ?= 6.0.0
 
-# Only used when "capella_loop.sh" is NOT used
-export DOCKER_TAG=$(CAPELLA_VERSION)-$(CAPELLA_DOCKERIMAGES_REVISION)
+# Comma-separated list of dropins to download & add, doesn't affect copied & mounted dropins
+# See available options in documentation.
```
Comment (dominik003) [inline Makefile:77]: Maybe add the link to the documentation page here

File: capella/install_dropins.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        check=True,
+    )
+
+
+if __name__ == "__main__":
+    dropins = load_dropins()
+    extract_repositories_and_installIUs(dropins)
+
+    # if repositories:
+    #     install_update_sites(repositories, install_ui)
```
Comment (dominik003) [inline capella/install_dropins.py:57]: I assume this should be removed

**Review by dominik003 — APPROVED** [review]
Looks good only a few small comments

File: Makefile (hunk 1 above)
Comment (MoritzWeber0) [reply Makefile:77]: Added.

File: Makefile [hunk 3]
```diff
@@ -350,8 +362,10 @@ run-t4c/client/exporter: t4c/client/exporter
 		-e HTTP_LOGIN="${HTTP_LOGIN}" \
 		-e HTTP_PASSWORD="${HTTP_PASSWORD}" \
 		-e LOG_LEVEL="$(LOG_LEVEL)" \
-		$(DOCKER_PREFIX)t4c/client/exporter:$(DOCKER_TAG)
+		$(DOCKER_PREFIX)t4c/client/exporter:$(echo "$(DOCKER_TAG_SCHEMA)" | envsubst)
 
+debug-capella/base: DOCKER_RUN_FLAGS=-it --entrypoint="bash"
```
[the remark this answers is not in the document]
Comment (MoritzWeber0) [reply Makefile:367]: Bash is shorter :)

**Pull request merged**